src/include/dns-server/dialog-main.rb

```diff
@@ -40,6 +40,9 @@ def initialize_dns_server_dialog_main(include_target)
       @initial_screen = "start_up"
 
       @service = SystemdService.find("named")
+
+      raise "Service 'named' not found. Please, install the required packages." unless @service
+
```

Comment:
That's my intention. This module is only used internally by dns-server client. And if you pretend to use from another place (I don't expect anyone to do that), you should make sure that the package is installed.